core/server/data/importer/importers/data/base.js

```diff
@@ -182,54 +182,57 @@ class Base {
 
         debug('afterImport', this.modelName);
 
-        return Promise.each(this.dataToImport, function (obj) {
-            if (!obj.model) {
-                return;
-            }
-
-            return Promise.each(['author_id', 'published_by', 'created_by', 'updated_by'], function (key) {
-                // CASE: not all fields exist on each model, skip them if so
-                if (!obj[key]) {
-                    return Promise.resolve();
-                }
-
-                oldUser = _.find(self.users, {id: obj[key]});
-
-                if (!oldUser) {
-                    self.problems.push({
-                        message: 'Entry was imported, but we were not able to update user reference field: ' + key,
-                        help: self.modelName,
-                        context: JSON.stringify(obj)
-                    });
-
-                    return;
-                }
-
-                return models.User.findOne({
-                    email: oldUser.email,
-                    status: 'all'
-                }, options).then(function (userModel) {
-                    // CASE: user could not be imported e.g. multiple roles attached
-                    if (!userModel) {
-                        userModel = {
-                            id: models.User.ownerUser
-                        };
+        return models.User.getOwnerUser(options)
```

Comment:
Nothing in this file has changed, except of fetching the owner user. This is not optimal, because it would fetch the owner for each table, but it should be a cached query. There is no need in my opinion to pre-optimise this until we want to optimise the performance of the importer.